packages/cxl-ui/src/components/cxl-star-rating.js

```diff
@@ -0,0 +1,64 @@
+import '@conversionxl/cxl-lumo-styles';
+import { IronStarRating } from '@cwmr/iron-star-rating';
+
+export class CxlStarRating extends IronStarRating {
```

Comment:
Missing all caps `CXL`.

Reply:
Please review all files for this. It seems like we could use an `eslint` rule to automate, but I don't have good experience in building one.

Reply:
sorry, camel-case habit, fixed in: https://github.com/conversionxl/aybolit/pull/88/commits/9bc60c6a811ebebc5f1851de2c0b92dca57c7bc1

Reply:
https://www.npmjs.com/package/generator-eslint - there's nice tool for creating `eslint` plugins. But this would require to dive into the documentation: https://eslint.org/docs/developer-guide/working-with-rules. Should I do that?

Reply:
> Should I do that?

Nah, I think too much of a side track if you also need to learn the details.

Unless you can somehow magically pull this off in 1 h?

Reply:
I also started thinking, if anything, this should be a Prettier rule, not eslint.